client/src/views/quizView.js

```diff
@@ -35,7 +35,14 @@ function disableRadioBtns(){
 
 function colourLabels(correctAnswer){
   [...document.getElementsByTagName('label')].forEach((label) => {
-    label.style.color = label.innerText == correctAnswer ? 'green':'grey';
+    var p = null;
```

Comment:
I'm confused what this function is doing now, am I being dense?

Reply:
I had to give the labels a p element of the answers instead of setting the label's inner text so that I could align it in css. Then I realised I'd broken the colour change. So I changed it so when it loops through the labels, instead of styling the label inner text I needed to find the p element that belongs to the label and style the p element's inner text. so it loops through the child elements of the label and finds the p element by class name. Does that make sense?

Reply:
Yes, thanks, and nice work